pkg/database/import.go

```diff
@@ -149,7 +149,7 @@ func (s *Importer) importTable(table string) error {
 				}
 				if err := destTX.
 					Table(table).
-					Clauses(clause.OnConflict{DoNothing: true}).
+					Clauses(clause.OnConflict{DoNothing: true}, clause.Returning{}).
```

Comment:
This moment fix the issue I mentioned on meeting which is related to `LastInsertId` here.